contrib/go-redis/redis/redis_test.go

```diff
@@ -313,3 +313,58 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.0, WithAnalyticsRate(0.0))
 	})
 }
+
+type testKey string
+
+func TestWithContextRace(t *testing.T) {
+	opts := &redis.Options{Addr: "127.0.0.1:6379"}
+	assert := assert.New(t)
+
+	k := testKey("key")
+	v := "val"
+	v2 := "val2"
```

Comment:
Do we really need these? We usually just repeat the strings everywhere. I understand it'd be nice to have them here if they were to change for any reason, but I can't think of any reason to change these except during the review process.

Reply:
Alright. Keep them if you like. I was wondering though: would they be better suited as constants? And would `v1` pair better with `v2` rather than `v` and `v1`? I might be a bit OCD 🙂

Reply:
I agree. `v` would be better called `v1`. I'm using them *exactly* like constants, so they probably should be. Apparently you can declare constants in a function, so maybe that would be the best way in the future to avoid polluting the rest of the file with constants. What do you think? I've gotten rid of them here anyway.

Reply:
Yeah, that's what I meant. I didn't mean to declare them globally.